cinder-{{cookiecutter.driver_name_lc}}/src/config.yaml

```diff
@@ -1,19 +0,0 @@
-options:
```

Comment:
I also think that this should stay. Isn't this a useful boilerplate example?

Reply:
it's now at the top-level directory. That's the correct place for operator framework charms, from what I could gather.

Reply:
I mean specifically `driver-source` and `driver-key`. I think this should stay (I don't see this in the top-level `/config.yaml` anymore).

https://github.com/openstack-charmers/cinder-storage-backend-template/pull/15/files/921b44fe7e1788b0d73f88ca0f25a92f96835330#diff-c2548685f2386708c10ef678f35983f20049ce01e8452a9e54fba1a9ff082f80

Reply:
Oh I understand: `OSBaseCharm` provides the same mechanism for free with `source` and `key`, thanks a lot!